README.rst

```diff
@@ -12,9 +12,40 @@
 .. image:: doc/source/_static/gnocchi-logo.png
 
 Gnocchi is an open-source, multi-tenant timeseries, metrics and resources
-database. It provides an `HTTP REST`_ interface to create and manipulate the
-data. It is designed to store metrics at a very large scale while providing
-access to metrics and resources information and history.
+database.
+
+The problem that Gnocchi solves is storage and indexing of time series data and
+resources at large scale, especiallly in the context of cloud computing. Modern
+cloud platforms are not only huge, but they also are dynamic and potentially
+multi-tenant. Gnocchi takes all of that into account.
+
+Gnocchi takes a unique approach to time series storage: rather than storing raw
+data points, it aggregates them before storing them. This built-in feature is
+different from most other time series databases, which usually support this
+mechanism as an option and compute aggregation (average, minimum, etc.) at
+query time.
+
+Because Gnocchi computes all the aggregations at ingestion, getting the data
+back is extremely fast, as it just needs to read back the pre-computed results.
+
+The way those data points are aggregated is configurable on a per-metric basis,
+using an archive policy. An archive policy defines which aggregations to
+compute and how many aggregates to keep. Gnocchi supports a wild number of
+aggregation methods, such as minimum, maximum, average, Nth percentile,
+standard deviation, etc. Those aggregations are computed over a period of time
+(called granularity) and are kept for a defined timespan. Aggregates are stored
+in a compressed format, ensuring the data points take as little space as
+possible.
+
+For example, imagine you define an archive policy to keep the average, minimum,
+and maximum of a time series with five-minute granularity for 30 days. In that
+case, Gnocchi will compute the average, minimum, and maximum of the values over
+five-minute ranges, keeping up to 8,640 points (the number of five-minute
+aggregates you can have over 30 days).
+
+It has been designed to handle large amount of |aggregates| being stored, while
```

Comment:
i imagine the readme has no concept of glossary so the '|' do nothing? that said, this is just repeating what you have earlier.

Reply:
the readme is included in `index.rst`